model/trainer.py

```diff
@@ -592,6 +596,10 @@ def __init__(
             ),
             out_features=1
         )
+        self.style_embedding = torch.nn.Embedding(
+            num_embeddings=3,
+            embedding_dim=self.config["embedding_dimension"]
+        )
 
         self.initialize_parameters()
```

Comment:
also need to update `initialize_parameters` when `style_embedding` is introduced?
right now, `style_embedding` is caught by `weight_regex` (and applied Xavier Normal init) in `initialize_parameters`, is it expected?

Reply:
Yeah, this is intended. That being said: For embeddings layers, weight initialization isn't super important. (As demonstrated here https://arxiv.org/pdf/1711.09160.pdf for instance.) Even initializing them as zero tensors should be fine.